zipline/data/loader.py

```diff
@@ -183,7 +213,7 @@ def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day,
         data.to_csv(get_data_filepath(filename, environ))
     except (OSError, IOError, HTTPError):
         logger.exception('Failed to cache the new benchmark returns')
-        raise
+        return None
```

Comment:
I'm not sure how I feel about returning `None` in the failure case here. In my experience, this kind of error handling tends to create more confusing failures, because when an error happens, it gets ignored and causes new errors later in code that wasn't expecting a return value of `None`.

If we want to keep this, we should at least document that this function returns `None` in the error, case, but I'm wondering if this is actually necessary if we've made it easy to disable the benchmark.